Umbraco-Cloud/Upgrades/Migrating-from-9-to-10/index.md

```diff
@@ -0,0 +1,130 @@
+## Step 2: Upgrade the project locally using Visual Studio
+
+* Open your project in Visual Studio.
+
+* Right-click your project solution in the **Solution Explorer** and select **Properties**.
+    ![Solution Explorer](images/Solution-Explorer.png)
+
+* In the **General** section of the **Application** tab, select **.Net 6.0** from the **Target Framework** drop-down.
+    ![Target Framework](images/Target-Framework.png)
+
+* Go to **Tools** > **NuGet Package Manager** > **Manage NuGet Packages for Solution...**.
+
+* In the **NuGet Package manager Solution**, go to the **Installed** tab and choose **Umbraco.Cms**.
+
+* Select your project, choose **10.0.0** from the **Version** drop-down and click **Install** to upgrade your project to version 10.
+    ![Nuget Version Install](images/Nuget-Version-Install.png)
+
+* Similarly, update the below packages referenced in your project to the latest versions available on NuGet either through the **NuGet Package Manager** or **Package Manager Console (PowerShell)**:
+  * <https://www.nuget.org/packages/Umbraco.Cms/10.0.0>
+  * <https://www.nuget.org/packages/Umbraco.Deploy.Cloud/10.0.0>
+  * <https://www.nuget.org/packages/Umbraco.Deploy.Contrib/10.0.0>
+  * <https://www.nuget.org/packages/Umbraco.Forms/10.0.0>
+  * <https://www.nuget.org/packages/Umbraco.Cloud.StorageProviders.AzureBlob/5.0.0>
+  * <https://www.nuget.org/packages/Umbraco.Cloud.Identity.Cms/10.0.0>
+  * <https://www.nuget.org/packages/Umbraco.Deploy.Forms/10.0.0/>
+
+* Install the [Microsoft.Extensions.DependencyInjection.Abstractions](https://www.nuget.org/packages/Microsoft.Extensions.DependencyInjection.Abstractions/6.0.0) package.
+
+* Update the `Program.cs` to the following:
+
+    ```CSharp
+    using Umbraco.Cms.Web.Common.Hosting;
+    public class Program
+        {
+            public static void Main(string[] args)
+                => CreateHostBuilder(args)
+                    .Build()
+                    .Run();
+
+            public static IHostBuilder CreateHostBuilder(string[] args) =>
+                Host.CreateDefaultBuilder(args)
+                    .ConfigureUmbracoDefaults()
+                    .ConfigureWebHostDefaults(webBuilder =>
+                    {
+                        webBuilder.UseStaticWebAssets();
+                        webBuilder.UseStartup<Startup>();
+                    });
+        }
+    ```
+
+* To re-enable the appsettings IntelliSense, update your schema reference in the **appsettings.json**, **appsettings.Development.json**, **appsettings.Production.json**, and **appsettings.Staging.json** files from:
+
+    ```json
+    "$schema": "./umbraco/config/appsettings-schema.json",
+    ```
+
+    To:
+
+    ```json
+    "$schema": "./appsettings-schema.json",
+    ```
+
+* Remove the following files and folders *manually* from your local project and on the cloud **Development** environment through [KUDU](../../Set-Up/Power-Tools/index.md) both from the `repository` and `wwwroot` folders:
```

Comment:
This is a long item.
Would it make sense to split it in two?
One for local, and one for KUDU?